topics/variant-analysis/tutorials/trio-analysis/tutorial.md

```diff
@@ -0,0 +1,441 @@
+---
+layout: tutorial_hands_on
+
+title: Trio Analysis using Synthetic Datasets from RD-Connect GPAP
+zenodo_link: 'https://sandbox.zenodo.org/record/1055250'
+questions:
+- How do you import data from the EGA?
+- How to download files with HTSGET in Galaxy?
+- How do you pre-process VCFs?
+- How do you identify causative variants?
+objectives:
+- Requesting DAC access and importing data from the EGA.
+- Pre-process VCFs using regular expressions and AWK.
+- Use annotations and phenotype information to find the causative variant(s).
+time_estimation: 2H
+key_points:
+- Downloading whole datasets with HTSGET is safe and easy with galaxy.
+- AWK and regex are usefull tools for pre-processing VCF files.
+- Variant annotations allows us to strictly filter VCFs to find the causative variant.
+contributors:
+- Jasper Ouwerkerk
+
+---
+
+# Introduction
+{:.no_toc}
+To discover causal mutations of inherited diseases it’s common practice to do a trio analysis. In a trio analysis DNA is sequenced of both the patient and parents. Using this method, it’s possible to identify multiple inheritance patterns. Some examples of these patterns are autosomal recessive, autosomal dominant, and  de-novo variants, which are represented in the figure below. To elaborate, the most left tree shows an autosomal dominant inhertitance pattern where the offspring inherits a faulty copy of the gene from one of the parents. The center subfigure represents an autosomal recessive disease, here the offspring inherited a faulty copy of the same gene from both parents. In the right subfigure a de-novo mutation is shown, which is caused by a mutation during the offspring’s lifetime.
+
+| ![plot](../../images/trio-analysis/pedigree.svg) |
+|:--:|
+| **Figure 1:** *3 family trees representing autosomal dominant, autosomal recessive, and a de-novo inheritence patter from parents to son.* |

```

Comment:
```suggestion
![please write a better plot description here](../../images/trio-analysis/pedigree.svg "3 family trees representing autosomal dominant, autosomal recessive, and a de-novo inheritence patter from parents to son")
```

This can be handled automatically (including the numbering) via this syntax, `![alt-text](image.ext "the figure caption you want displayed")`

It also looks like this alt-text doesn't describe the image very well for users who are relying on a screen reader to access the content. We have a short guide on [what's appropriate for alt-text vs captions in the GTN Contributing Guide](https://training.galaxyproject.org/training-material/topics/contributing/tutorials/create-new-tutorial-content/tutorial.html#guidelines-on-alt-vs-figcaption-text) which has more details.

Reply:
Should be fixed in: c5727787e1bbd8eab2fe9c6b9be01bee010f7132